### ISSUE TITLE
Update the cli mapping for extension install command

### ISSUE THREAD

**Issue by soninaren** [Azure/azure-functions-core-tools#636]
We need to update the extension package mapping to latest packages. Also need to add storage extension to the mapping.

**Comment by ahmelsayed** [comment]
You already did this, right?

**Comment by soninaren** [comment]
Yes, But i forgot to put do not merge label on it. We will have to do another update when the packages are released to NuGet.

**Comment by ahmelsayed** [comment]
okay, will keep this open then
